00:00:00.002871 main     VirtualBox COM Server 7.0.4 r154605 win.amd64 (Nov 16 2022 15:33:13) release log
00:00:00.002872 main     Log opened 2023-01-03T07:09:01.361056600Z
00:00:00.002873 main     Build Type: release
00:00:00.002878 main     OS Product: Windows 11
00:00:00.002878 main     OS Release: 10.0.22000
00:00:00.002879 main     OS Service Pack: 
00:00:00.027156 main     DMI Product Name: 82H8
00:00:00.040134 main     DMI Product Version: IdeaPad 3 15ITL6
00:00:00.040145 main     Firmware type: UEFI
00:00:00.040409 main     Secure Boot: VERR_PRIVILEGE_NOT_HELD
00:00:00.040428 main     Host RAM: 7991MB (7.8GB) total, 384MB available
00:00:00.040431 main     Executable: C:\Program Files\Oracle\VirtualBox\VBoxSVC.exe
00:00:00.040432 main     Process ID: 107368
00:00:00.040432 main     Package type: WINDOWS_64BITS_GENERIC
00:00:00.467905          VirtualBox: object creation starts
00:00:00.468631          Home directory: 'C:\Users\harsh\.VirtualBox'
00:00:00.468860          Installed Drivers:
00:00:00.474629            C:\Windows\system32\DRIVERS\VBoxSup.sys (Version: 7.0.4.4605)
00:00:00.475108            C:\Windows\system32\DRIVERS\VBoxUSBMon.sys (Version: 7.0.4.4605)
00:00:00.475565            C:\Windows\system32\DRIVERS\VBoxNetAdp6.sys (Version: 7.0.4.4605)
00:00:00.475986            C:\Windows\system32\DRIVERS\VBoxNetLwf.sys (Version: 7.0.4.4605)
00:00:00.476506          Loading settings file "C:\Users\harsh\.VirtualBox\VirtualBox.xml" with version "1.12-windows"
00:00:00.478716          usbLibGetDevices: Starting USB device enumeration
00:00:00.516577          Getting USB descriptor (id 0) failed (dwErr=31) on hub USB#ROOT_HUB30#4&3881863d&0&0#{f18a0e88-c30c-11d0-8815-00a0c906bed8} port 5
00:00:00.516629          Getting USB descriptor (id 0) failed (dwErr=31) on hub USB#ROOT_HUB30#4&3881863d&0&0#{f18a0e88-c30c-11d0-8815-00a0c906bed8} port 10
00:00:00.518846          usbLibGetDevices: Found 2 USB devices, 0 captured
00:00:00.518911          USB: Unknown USB device detected (idVendor: 0x0bda, idProduct: 0xc123)
00:00:00.518958          USB: Unknown USB device detected (idVendor: 0x174f, idProduct: 0x2459)
00:00:00.634024          HostDnsMonitor: initializing
00:00:00.641354          HostDnsMonitor: updating information
00:00:00.641383          HostDnsMonitor: old information
00:00:00.641384            no server entries
00:00:00.641384            no domain set
00:00:00.641385            no search string entries
00:00:00.641385          HostDnsMonitor: new information
00:00:00.641385            server 1: 192.168.1.1
00:00:00.641386            domain: home
00:00:00.641386            no search string entries
00:00:00.695526          VD: VDInit finished with VINF_SUCCESS
00:00:00.697122          Loading settings file "C:\Users\harsh\VirtualBox VMs\ISE_14.7_VIRTUAL_MACHINE\ISE_14.7_VIRTUAL_MACHINE.vbox" with version "1.19-windows"
00:00:00.698007          VirtualBox: object created
00:00:00.798615          Launched VM: 1217303264 pid: 99772 (0x185bc) frontend: GUI/Qt name: ISE_14.7_VIRTUAL_MACHINE
00:00:07.502077          ERROR [COM]: aRC=VBOX_E_OBJECT_NOT_FOUND (0x80bb0001) aIID={70401eef-c8e9-466b-9660-45cb3e9979e4} aComponent={ExtPackManagerWrap} aText={No extension pack by the name 'Oracle VM VirtualBox Extension Pack' was found}, preserve=false aResultDetail=0
00:00:12.118665          ERROR [COM]: aRC=VBOX_E_INVALID_VM_STATE (0x80bb0002) aIID={300763af-5d6b-46e6-aa96-273eac15538a} aComponent={SessionMachine} aText={The machine is not mutable or saved (state is Restoring)}, preserve=false aResultDetail=0
00:00:12.477815          NetworkServiceRunning: started 'VBoxNetDHCP.exe', pid 00019fcc
00:00:13.735342          ERROR [COM]: aRC=VBOX_E_OBJECT_NOT_FOUND (0x80bb0001) aIID={ad47ad09-787b-44ab-b343-a082a3f2dfb1} aComponent={MediumWrap} aText={Property 'CRYPT/KeyId' does not exist}, preserve=false aResultDetail=0
00:00:13.736786          Added USB filter (ID=1, type=4) for device 03FD:FFFFFFFF rev FFFFFFFF, c/s/p FFFFFFFF/FFFFFFFF/FFFFFFFF, Manufacturer=`<null>' Product=`<null>' Serial=`<null>'
00:00:13.736840          Added USB filter (ID=2, type=4) for device 0403:FFFFFFFF rev FFFFFFFF, c/s/p FFFFFFFF/FFFFFFFF/FFFFFFFF, Manufacturer=`<null>' Product=`<null>' Serial=`<null>'
00:00:13.736847          Added USB filter (ID=3, type=4) for device 1443:FFFFFFFF rev FFFFFFFF, c/s/p FFFFFFFF/FFFFFFFF/FFFFFFFF, Manufacturer=`<null>' Product=`<null>' Serial=`<null>'
00:00:17.634014          Saving settings file "C:\Users\harsh\VirtualBox VMs\ISE_14.7_VIRTUAL_MACHINE\ISE_14.7_VIRTUAL_MACHINE.vbox" with version "1.19-windows"
00:00:17.636979          Finished saving settings file "C:\Users\harsh\VirtualBox VMs\ISE_14.7_VIRTUAL_MACHINE\ISE_14.7_VIRTUAL_MACHINE.vbox"
00:10:00.561189 USBPROXY usbLibGetDevices: Starting USB device enumeration
00:10:00.591727 USBPROXY Getting USB descriptor (id 0) failed (dwErr=31) on hub USB#ROOT_HUB30#4&3881863d&0&0#{f18a0e88-c30c-11d0-8815-00a0c906bed8} port 5
00:10:00.591907 USBPROXY Getting USB descriptor (id 0) failed (dwErr=31) on hub USB#ROOT_HUB30#4&3881863d&0&0#{f18a0e88-c30c-11d0-8815-00a0c906bed8} port 10
00:10:00.592528 USBPROXY usbLibGetDevices: Found 2 USB devices, 0 captured
01:06:31.093342 USBPROXY usbLibGetDevices: Starting USB device enumeration
01:06:31.112097 USBPROXY Getting USB descriptor (id 0) failed (dwErr=31) on hub USB#ROOT_HUB30#4&3881863d&0&0#{f18a0e88-c30c-11d0-8815-00a0c906bed8} port 10
01:06:31.114634 USBPROXY usbLibGetDevices: Found 2 USB devices, 0 captured
01:55:35.766789          Removed USB filter ID=1
01:55:35.767057          Removed USB filter ID=2
01:55:35.767065          Removed USB filter ID=3
01:55:35.969584          Saving settings file "C:\Users\harsh\VirtualBox VMs\ISE_14.7_VIRTUAL_MACHINE\ISE_14.7_VIRTUAL_MACHINE.vbox" with version "1.19-windows"
01:55:35.977423          Finished saving settings file "C:\Users\harsh\VirtualBox VMs\ISE_14.7_VIRTUAL_MACHINE\ISE_14.7_VIRTUAL_MACHINE.vbox"
01:55:36.218141 SaveState Saving settings file "C:\Users\harsh\VirtualBox VMs\ISE_14.7_VIRTUAL_MACHINE\ISE_14.7_VIRTUAL_MACHINE.vbox" with version "1.19-windows"
01:55:36.224863 SaveState Finished saving settings file "C:\Users\harsh\VirtualBox VMs\ISE_14.7_VIRTUAL_MACHINE\ISE_14.7_VIRTUAL_MACHINE.vbox"
01:55:42.506548 main     VirtualBox: object deletion starts
01:55:42.506615 main     ERROR [COM]: aRC=VBOX_E_OBJECT_IN_USE (0x80bb000c) aIID={ad47ad09-787b-44ab-b343-a082a3f2dfb1} aComponent={MediumWrap} aText={Medium 'C:\Users\harsh\VirtualBox VMs\ISE_14.7_VIRTUAL_MACHINE\14.7_VM-disk001.vmdk' cannot be closed because it is still attached to 1 virtual machines}, preserve=false aResultDetail=0
01:55:42.509120 main     NetworkServiceRunning: killing VBoxNetDHCP.exe, pid 00019fcc...
01:55:42.521171 main     HostDnsMonitor: shutting down ...
01:55:42.521352 main     HostDnsMonitor: shut down
01:55:42.527551 Watcher  ERROR [COM]: aRC=E_ACCESSDENIED (0x80070005) aIID={7682d5eb-f00e-44f1-8ca2-99d08b1cd607} aComponent={VirtualBoxWrap} aText={The object is not ready}, preserve=false aResultDetail=0
01:55:42.529234 main     VirtualBox: object deleted
